Composer/packages/extensions/visual-designer/src/schema/uischema.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { SDKTypes } from '@bfc/shared';
+
+import { ActionCard } from '../widgets/ActionCard';
+import { generateSDKTitle } from '../components/nodes/utils';
+
+import { UISchema } from './uischema.types';
+
+export const uiSchema: UISchema = {
+  [SDKTypes.EditArray]: {
+    'ui:widget': ActionCard,
+    title: generateSDKTitle(),
```

Comment:
It seems like this could be the default instead of the "override".

Reply:
I want to add some settings in uischema to avoid duplications, what about `defaultWidgetProps`?
```ts
export const uischema = {
    'defaultWidgetProps': {
        'ActionCard': {
            'title': data => getSDKTitle(data)
        }
    },
    [SDKTypes.SetProperty]: {
        'ui:widget': 'ActionCard',
        content: data => data.property
    }
}
```

Reply:
What I'm saying is that we aren't really override any value. The UISchema is for overrides. But `'ui:widget': 'ActionCard'` is already doing the override. So it's fair to me to have the `getSDKTitle` in the ActionCard component since that is the default for that custom widget. Make sense? So in order to override an ActionCard's title, you could pass in another value for title in the UISchema.

Reply:
That's the right way, I got it

Reply:
resolved as Andy suggested